### PR TITLE
pulumi-new: add page

### PR DESCRIPTION
**Pull request by Denis-Kuso**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/pulumi-new.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+`pulumi new {{path/to/templates/aws-typescript}}`
+
+- Create a project from a specific source control location:
+
+`pulumi new {{https://github.com/user/repo}}`
+
+- Use the "passphrase" secrets provider with the pulumi.com backend:
+
+`pulumi new --secrets-provider=passphrase`
```
Comment (gutjuri) [inline pages/common/pulumi-new.md:24]:
```suggestion
- Use the specified secrets provider with the pulumi.com backend:

`pulumi new --secrets-provider={{passphrase}}`
```

File: pages/common/pulumi-new.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+- Create a project from a local file:
+
+`pulumi new {{path/to/templates/aws-typescript}}`
+
+- Create a project from a specific source control location:
+
+`pulumi new {{https://github.com/user/repo}}`
+
+- Use the specified secrets provider with the pulumi.com backend:
```
Comment (kbdharun) [inline pages/common/pulumi-new.md:22]:
```suggestion
- Use the specified secrets provider with the <pulumi.com> backend:
```
We enclose links inside angle brackets to be rendered by our clients.

File: pages/common/pulumi-new.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# pulumi new
+
+> Create a new Pulumi project.
+> More information: <https://www.pulumi.com/docs/iac/cli/commands/pulumi_new/>.
+
+- Choose a template interactively:
+
+`pulumi new`
+
+- Create a project from a specific template (e.g azure-python):
```
Comment (kbdharun) [inline pages/common/pulumi-new.md:10]:
```suggestion
- Create a project from a specific template (e.g `azure-python`):
```

**Review by kbdharun — COMMENTED** [review]
Welcome to tldr and thanks for your contribution. I have some minor suggestions for this page after which the PR is GTG.

---

**Note**. I closed and reopened the PR to retrigger the frozen CLA check.

File: pages/common/pulumi-new.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+`pulumi new {{provided-template}}`
+
+- Create a project from a local file:
+
+`pulumi new {{path/to/templates/aws-typescript}}`
+
+- Create a project from a specific source control location:
+
+`pulumi new {{https://github.com/user/repo}}`
```
Comment (spageektti) [inline pages/common/pulumi-new.md:20]: Instructions unclear. Does it accept only github links, or any Git repo, or can I use SVN too?

```suggestion
- Create a project from a Git repository:

`pulumi new {{url}}`
```

**Review by spageektti — APPROVED** [review]
Thanks for your contribution.

File: pages/common/pulumi-new.md (hunk 2 above)
Comment (spageektti) [inline pages/common/pulumi-new.md:22]:
```suggestion
- Use the specified secrets provider with the Pulumi backend:
```

File: pages/common/pulumi-new.md (hunk 4 above)
Comment (Denis-Kuso) [reply pages/common/pulumi-new.md:20]: Good point. Any git repo will do, private or hosted on some public site.

**Review by spageektti — APPROVED** [review]

**Pull request merged**